aim/sdk/objects/plugins/dvc_metadata.py

```diff
@@ -27,16 +39,28 @@ def __init__(self, url='.', path=None, rev=None, recursive=False, dvc_only=False
                     recursive=recursive,
                     dvc_only=dvc_only
                 )
-            )
+            ),
         }
 
     def _get_dvc_tracked_files(self, **ls_kwargs):
-        try:
-            from dvc.repo import Repo
-        except ImportError:
-            raise RuntimeError(
-                'Could not import module dvc.'
-            )
-
         entries = Repo.ls(**ls_kwargs)
         return [entry['path'] for entry in entries]
+
+    def _get_dvc_params(self, url):
+        try:
+            params = api.params_show(repo=url)
+            return params
+        except Exception:
+            logging.warning("Failed to log params")
+
+    def _get_dvc_lock(self, url):
+        try:
+            with open(Path(url).joinpath('dvc.lock'), 'r') as f:
+                try:
+                    content = yaml.safe_load(f)
+                    return content
+                except yaml.YAMLError as exc:
+                    logging.warning(exc)
+                content = f.readlines()
+        except FileNotFoundError:
+            raise FileNotFoundError(f"Failed to find dvc.lock in the repo {url}")
```

Comment:
what about logging a warning instead of re-raising the exception? it will allow to continue the execution; we can set the 'version' field to 'N/A' in that case.

Reply:
similar to the get_dvc_params method